web/css/ToDoListStyleSheet.css

```diff
@@ -39,7 +42,7 @@ body{
     height: 70%;
 }
 .toDoListStyle {
-    overflow-x: hidden;
+    overflow-y : hidden;
```

Comment:
i think I removed this yesterday. this should be overflow-x. you changed it to overflow-y.